ec2/keypair.py

```diff
@@ -89,6 +91,9 @@ def create(**kwargs):
         kp.name, ctx.instance, external=False)
     _save_key_pair(kp)
 
+    ctx.instance.runtime_properties[constants.AWS_TYPE_PROPERTY] = \
```

Comment:
make sure unassign_runtime_property_from_resource removes this runtime property on deletion